src/shadowfax.py

```diff
@@ -257,7 +255,7 @@ async def shadowfax(message, guid, logger):
                 single_edges, single_support_graphs, single_nodes = set(), set(), set()
                 # get nodes and edges from path
                 try:
-                    single_edges, single_support_graphs, single_nodes = recursive_filter_edge_support_graphs(
+                    single_edges, single_support_graphs, single_nodes = recursive_maintain_edge_support_graphs(
```

Comment:
To be knit-picky, I don't think this method name is still indicative of what it does. It does not maintain anything. It "gets" edges, nodes, and aux graphs and returns them. It's then up to the calling function to either maintain or filter or whatever it wants to do.